x-pack/legacy/plugins/snapshot_restore/server/routes/api/app.ts

```diff
@@ -68,8 +68,8 @@ export const getPermissionsHandler: RouterRouteHandler = async (
   });
 
   // Find missing cluster privileges and set overall app permissions
-  permissionsResult.missingClusterPrivileges = extractMissingPrivileges(cluster || {});
-  permissionsResult.hasPermission = hasPermission;
+  privilegesResult.missingPrivileges.cluster = extractMissingPrivileges(cluster);
+  privilegesResult.hasAllPrivileges = hasPermission;
```

Comment:
Did you want to change this to privileges too? I did a ctrl-f for "permission" on the diff of this PR and there are also quite a few references to privileges in the localization keys.

Reply:
Not sure I follow here. Yes I did want to change "permission" to "privileges" here. Isn't that what you also wanted?

Reply:
Ah you mean the "hasPermission" name? 😊

Reply:
> Ah you mean the "hasPermission" name

Yes, you can do a ctrl-f and you'll see all of the instances I have in mind as candidates for renaming.

Reply:
++ I just updated everything that had the word "permission".